### PR TITLE
add image component

### PR DESCRIPTION
**Pull request by shubhamkumar648**
various image
1. responsive image
2. round image
img

### CODE REVIEW

File: styles/image.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.Image__container{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    text-align: center;
```
Comment (srejitk) [inline styles/image.css:5]: This can be extracted to another utility class

File: styles/image.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    height: 20rem;
+    object-fit: cover;
+    border-radius: 50%;
+}
+
+.code_snippet_display{
+    width: 100%;
+    max-height: 13.3125rem;
+    padding: 1rem 0;
+    background-color: rgb(250, 250, 250);
```
Comment (srejitk) [inline styles/image.css]: Is this a single instance, if it's common please use a variable

**Review by srejitk — COMMENTED** [review]
Suggesting some minor changes.

File: styles/image.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.Image__container{
```
Comment (krishanu2911) [inline styles/image.css]: maybe there is a typo Image make it small "i" it may create problem in future

**Review by krishanu2911 — COMMENTED** [review]
Well written code no changes required just added a few suggestions.

File: styles/image.css (hunk 1 above)
Comment (shubhamkumar648) [reply styles/image.css:5]: thank you for consider! i will do it later during refactor code

File: styles/image.css (hunk 3 above)
Comment (shubhamkumar648) [reply styles/image.css]: thank you for aknowledge , i will update in next commit

**Review by krishanu2911 — APPROVED** [review]

**Review by srejitk — APPROVED** [review]

**Comment by shubhamkumar648** [comment]
> Well written code no changes required just added a few suggestions.

thank you

**Pull request merged**